apps/website/src/broken-link-handler.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+import React, { PropsWithChildren } from 'react';
+import { ErrorBoundary } from 'react-error-boundary';
+
+export function BrokenLinkHandler({ children }: PropsWithChildren) {
+  return (
+    <ErrorBoundary
+      onError={(error) => {
+        if ((error as any).statusCode === 404) {
+          window.location.reload();
```

Comment:
Aha, so this is the reason of https://waku.amazeelabs.dev/en/content-hub craziness 😅

Reply:
Yes, this has to become more solid.